openslides/core/static/js/core/core.js

```diff
@@ -605,6 +605,46 @@ angular.module('OpenSlidesApp.core.site', [
             }
         );
     };
+
+    // *** countdown ***
+    $scope.countdowns = [];
+    $scope.messages = [];
+    _.forEach(Projector.get(1).elements, function(element) {
+        if (element.name == 'core/countdown') {
+            $scope.countdowns.push(element);
+        }
+        if (element.name == 'core/message') {
+            $scope.messages.push(element);
+        }
+    });
+    $scope.addCountdown = function () {
+        $http.post(
+            '/rest/core/projector/1/activate_elements/',
+            [{name: 'core/countdown', status: 'stop', visible: true,
+                countdown_time: 42, stable: false}]
+            // TODO countdown_time: config auslesen
+        ).success(function(data) {
+            $scope.countdowns = [];
+            _.forEach(data.elements, function(element) {
+                if (element.name == 'core/countdown') {
+                    $scope.countdowns.push(element);
+                }
+            });
+        });
+    };
+    $scope.removeCountdown = function (countdown) {
+        // TODO
+    };
+    var stopped;
+    $scope.startCountdown = function (countdown) {
+        stopped = $timeout(function() {
+            countdown.countdown_time--;
+            $scope.startCountdown(countdown);
+        }, 1000);
+    };
+    $scope.stopCountdown = function(){
+       $timeout.cancel(stopped);
+    };
```

Comment:
The scenario is like this:

* The projector entry says "status": "stop". This means the client should not have any `$timeout` loop running.
* Then someone clicks an start button.
* The client sends one request to the server. That's all. Don't start any `$timeout` loop.
* A while later the server sends an autoupdate info to the client telling them that "status" is now "running". This means the client should have such a `$timeout` loop running until it gets another autoupdate info.